doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -25,15 +25,143 @@
+| `req.inf.ntw.06` | Network | The Architecture **should** embrace open-based standards and technologies. |
+| `req.inf.ntw.09` | Network | The Architecture **must** be capable of supporting highly available (Five 9’s or better) CNF/VNF workloads. |
+| `req.inf.ntw.10` | Network | The Architecture **should** be architected to provide a standardised, scalable, and repeatable deployment model across all applicable NFVI sites. |
+| `req.inf.ntw.11` | Network | The SDN solution **should** be configurable in an automated manner using openly published API definitions. |
+| `req.inf.ntw.12` | Network | The SDN solution **should** be able to be centrally administrated and configured. |
+| `req.inf.ntw.13` | Network | The Architecture **must** support dual stack IPv4 and IPv6 in Pods and Services. |
+| `req.inf.ntw.14` | Network | The Architecture **must** support Conatiner Network Interfgace (CNI). |
+| `req.inf.acc.01` | Acceleration | The Architecture **should** support Application Specific Acceleration. |
+| `req.inf.acc.02` | Acceleration | The Architecture **should** support NFVI Acceleration (such as SmartNICs). |
+| req.inf.vir.01   | Virtualisation |   The Architecture **must** support the capability for Containers to consume virtualised compute, storage and network resources.|
+| req.inf.phy.01  |  Physical Infrastructure |   The Architecture **must** support the capability for Containers to consume physical (aka bare metal) compute, storage and network resources. |
 
+<p align="center"><b>Table 2-2:</b> Containerised Platform: Infrastructure Requirements.</p>
+Please note that "shared" is a reference to multi-tenant support and pooled storage resources.
+
+<a name="2.3.3"></a>
+### 2.3.3 Containerised Infrastructure Management (CIM)
+
+| Ref # | sub-category | Description |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `req.cim.01` | General | The Architecture **must** allow infrastructure resource sharing. |
+| `req.cim.02` | General | The Architecture **must** allow discovery and management of NFVI resources. |
+| `req.cim.03` | General | The Architecture **must** support Enhanced Platform Awareness (EPA). |
+| `req.cim.04` | General | The Architecture **must** include image repository management. |
+| `req.cim.05` | General | The Architecture **should** support a multi-tenanted environment. |
+| `req.cim.06` | General | The Architecture **must** support resource tagging. |
+| `req.cim.07` | General | The Architecture **must** support horizontal scaling. |
+| `req.cim.08` | General | The Architecture **must** support workload resiliency. |
+
+<p align="center"><b>Table 2-3:</b> Containerised Platform: CIM Requirements.</p>
```

Comment:
Suggest the table title is changed to "Kubernetes Architecture: Kubernetes Cluster Requirements"

Reply:
Thanks -- change made